pkg/bytecode/compiler.go

```diff
@@ -49,6 +49,8 @@ func (c *Compiler) Compile(node parser.Node) error {
 		return c.compileAssignment(node)
 	case *parser.BinaryExpression:
 		return c.compileBinaryExpression(node)
+	case *parser.GroupExpression:
+		return c.Compile(node.Expr)
```

Comment:
not sure if its worth leaving a comment to highlight this, it essentially just unwraps grouped expressions from the AST, which takes care of the ordering. I don't think that's very clear from what we have here, but I also couldn't think of good words to put here to explain it, suggestions welcome!

Reply:
I think this is fine as is. There's not much else to do with a grouped expression but to pass through to the contained expression. The "group" part of the grouped expression has no presence in the bytecode, which is the only thing I think that might be worth saying, but I don't really think that is necessary either.